.github/workflows/ci.yaml

```diff
@@ -1,16 +1,37 @@
 name: Quality
-on: push
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
 jobs:
   lint:
     name: Lint
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
-      - name: Use Node.js 12.x
-        uses: actions/setup-node@v1
+      - name: Cancel Previous Runs
+        if: github.ref != 'refs/heads/master'
+        uses: styfle/cancel-workflow-action@0.8.0
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Cache Node Modules
```

Comment:
For consistency, I'd recommend either renaming to `Cache node_modules` OR (possibly preferred?) renaming the step I created below from `Install node_modules` to `Install Node Modules`

Reply:
I copied this from m, not opposed to changing stuff tho https://github.com/cdr/m/blob/82e57ae7c0d35b3ececcd04446f8b9e617795d2a/.github/workflows/coder.yaml#L32

I think nobody will look at these unless stuff is broken, so have not given it much thought tbh lol